app/gui/view/graph-editor/src/lib.rs

```diff
@@ -2426,14 +2426,49 @@ fn new_graph_editor(app: &Application) -> GraphEditor {
     }
 
 
+    // === Add Node ===
+
+    frp::extend! { network
```

Comment:
This FRP piece is just moved to make `has_detached_edge` being available for `Node Editing` section.